.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,31 +4,41 @@ about: Create a report to help us improve
 
 ---
 
-**Affected Applications**
-Which app(s) does this bug affect?
+**Affected Product**
+Which product(s) does this bug affect?
+<!-- Pick one: Check-Ins, Giving, Groups, People, Services, Resources, Webhooks -->
```

Comment:
We should probably remove the `(s)` from the line above if we only want them to choose one.